src/Config.php

```diff
@@ -1201,4 +1138,58 @@ public function getCurrentTimestamp()
 
         return $now;
     }
+
+    /**
+     * Use get('general/database') instead
+     *
+     * @deprecated Since 2.1, to be removed in 3.0.
+     *
+     * @return array
+     */
+    public function getDBOptions()
+    {
+        return $this->get('general/database');
+    }
+
+    /**
+     * Use {@see Zone} instead with a {@see Request}.
+     *
+     * If you do not have a request then YDIW. Code should be agnostic
```

Comment:
While this is 100% correct, it's unfair to tell people  that are new(ish) to the game that following our advice a week ago was great, but now "you're doing it wrong"… just kinda feels brash.

Otherwise… code looks good, just the comment needs toning down though… Oh ane I'm the only perfect one here… well. me and the :koala: